src/styles/index.scss

```diff
@@ -1,3 +1,125 @@
+@import "/src/styles/variables";
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+.card {
+  display: flex;
+  width: 198px;
+  height: 406px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside use padding: 32px 16px 16px instead